module/connection/ServerDialog.kt

```diff
@@ -197,15 +203,26 @@ object ServerDialog {
     }
 
     @Composable
-    private fun CACertificateFormField(state: ConnectServerForm) {
+    private fun CACertificateFormField(state: ConnectServerForm, dialogWindow: ComposeDialog) {
         Field(label = Label.CA_CERTIFICATE) {
             TextInput(
                 value = state.caCertificate,
                 placeholder = "${Label.PATH_TO_CA_CERTIFICATE} (${Label.OPTIONAL.lowercase()})",
                 onValueChange = { state.caCertificate = it },
                 enabled = StudioState.client.isDisconnected,
-                modifier = Modifier.fillMaxSize(),
+                modifier = Modifier.weight(1f)
```

Comment:
This `TextInput` has to share the `Field` with the `IconButton`, so it can't just fill max size anymore.